components/calendar/calendar.js

```diff
@@ -594,15 +606,14 @@ class Calendar extends LocalizeCoreElement(RtlMixin(LitElement)) {
 		const date = selectedDate.getAttribute('data-date');
 
 		this.selectedValue = formatDateInISO({ year: year, month: (parseInt(month) + 1), date: date });
+		this._dateSelected = true;
 
 		const eventDetails = {
 			bubbles: true,
 			composed: true,
 			detail: { date: this.selectedValue }
 		};
 		this.dispatchEvent(new CustomEvent('d2l-calendar-selected', eventDetails));
-		await this._updateFocusDateOnChange();
```

Comment:
This was moved to be in `_updateFocusDateOnChange` since it was duplicated effort with `updated`